int32[] points
bool object_in_frame
